pkg/master/thirdparty/thirdparty.go

```diff
@@ -207,6 +207,9 @@ func (m *ThirdPartyResourceServer) getExistingThirdPartyResources(path string) [
 				Name:       key,
 				Namespaced: true,
 				Kind:       obj.Kind(),
+				Verbs: metav1.Verbs([]string{
+					"delete", "deletecollection", "get", "list", "patch", "create", "update", "watch",
```

Comment:
which is done for non-3rdparty resources, but not in a nicely factory way we can easily reuse. Not sure it's worth the effort here.